omusubi.d.ts

```diff
@@ -0,0 +1,67 @@
+declare module 'omusubi-js' {
+  class RawHTMLTagFuncOutput {
+    strings: string[];
+    values: string[];
+
+    constructor(strings: string[], values: string[]);
+  }
+  
+  export function defineComponent(tagname: string, options?: ElementDefinitionOptions): (wrapped: typeof Component) => void;
+
+  export function html(strings: string[], values: string[]): new () => RawHTMLTagFuncOutput;
+
+  export class Dispatcher extends Function {
+    constructor(element: unknown, store: unknown);
+    
+    dispatch(action: unknown, ...args: string[]): Promise<void>;
```

Comment:
"action" is a **string** which is name of the Dispatchers method.

Reply:
Changed :+1: